### DISCUSSION TITLE
cn/2024/08/09/xhs/

### DISCUSSION THREAD

**Discussion by giscus** [hongtaoh/hongtaoh.github.io#172] [Announcements]
# cn/2024/08/09/xhs/

https://hongtaoh.com/cn/2024/08/09/xhs/

**Comment by CyrusYip** [comment]
> 新账号没有什么个人信息，所以推送算法我觉得应该是符合大众的，而不是信息茧房 (echo chamber) 过的。

用不着个人信息，应用可以检测你的行为。就算一开始给你推送符合大众的内容，你在某些内容停留或者点进去看了，它就可以去猜你喜欢看这些。推送多一些又可以检测你的行为，看你对那些内容的反应。

**Reply by hongtaoh** [reply]
确实，不过这个新帐号我很少点开东西，可能加起来用了不到3小时这样。
